packages/libs/deer/Cargo.toml

```
@@ -20,7 +22,7 @@ rustc_version = "0.4.0"
 
 [features]
 default = ['std']
-std = ['serde/std']
+std = ['serde/std', 'spin/rwlock']
```

I think this is reversed 🙂↔️🙃  
```suggestion
std = ['serde/std']
```